packages/suite/src/hooks/wallet/useCoinmarketRecomposeAndSign.ts

```diff
@@ -24,6 +24,7 @@ export const useCoinmarketRecomposeAndSign = () => {
             ethereumDataHex?: string,
             recalcCustomLimit?: boolean,
             ethereumAdjustGasLimit?: string,
+            options: FormOptions[] = ['broadcast'],
```

Comment:
nitpick: if you put it on let's say fifth place, you can save the ugly `undefined` params

Reply:
I didn't want to interfere to other calls of this function, so I added it as last optional param.